x-pack/plugins/cloud_security_posture/public/common/types.ts

```diff
@@ -14,6 +14,11 @@ export type FindingsGroupByKind = 'default' | 'resource';
 export interface FindingsBaseURLQuery {
   query: Query;
   filters: Filter[];
+  /**
+   * Filters that are part of the query
+   * but not persisted in the URL or in the Filter Manager
+   */
+  additionalFilters?: Filter[];
```

Comment:
`notPersistedFilters` ?

Reply:
I don't like much `additionalFilters`, but I'm also not sure about `notPersistedFilters`, maybe `privateFilters` or `internalFilters`?

Reply:
I think either would do, `internalFilters` or `nonPersistedFilters`

Reply:
sticking with `nonPersistedFilters`, thanks for your input on that folks 🙌